packages/langium/src/typification/inferred-types.ts

```diff
@@ -0,0 +1,405 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import _ from 'lodash';
```

Comment:
This is not an explicit dependency of this package. Let's keep dependencies at a minimum and avoid the usage of additional libraries when possible.

Reply:
Note: it's ok to use lodash and other libraries in `langium-cli` because that is a CLI, not a library.